lib/sycamore/sycamore/utils/aryn_config.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import yaml
+import logging
+
+_DEFAULT_PATH = os.path.join(pathlib.Path.home(), ".aryn", "config.yaml")
+
+
+class ArynConfig:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
```

Comment:
Where do we use this cleverness?

I'd worry about the namespace overlap.  What if someone passes a kwarg called 'get_aryn_api_key'?

Reply:
Removed.